packages/amplify-category-api/src/graphql-transformer/transform-graphql-schema-v2.ts

```diff
@@ -258,6 +274,21 @@ const _buildProject = async (context: $TSContext, opts: TransformerProjectOption
   }
 };
 
+const isSqlLambdaVpcConfigRequired = async (
```

Comment:
N.p. but this method name suggests I'll get a boolean back, not a 'something or nothing'. Maybe rename to `getSqlDataSourceLambdaVpcConfig` ?